### ISSUE TITLE
Add Google Analytics tracking code to the episode popup links

### ISSUE THREAD

**Issue by michiecat** [a8cteam51/onbeing#95]
From Trent:

> The pop-up window that displays when users click on the play audio links doesn't seem to be tracking now, so we're not able to assess those events in terms of actions or pageviews, time spent on page, etc. Would it be possible to make sure we could add that tracking code to the pop-up instance so we can gain better insights into what our users are doing?

I'm working on a JavaScript solution for this, and will update here as I'm working. I'll have an update by Thursday, June 29th.

**Comment by ChrisHardie** [comment]
I resolved this by:

* Adding the Google Analytics code (defined in the ACF admin) into the HTML that is defined in the popup JS function - this will record a pageview when the popup loads
* Adding a manual GA event push to the onClick definition - this will record a [custom event](https://developers.google.com/analytics/devguides/collection/analyticsjs/events) that should be useful in tracking those clicks, even if the popup isn't displayed

I applied this to the "Play Episode" setup on the front page as well as the "Play Episode" and "Play Unedited" links on the single episode pages.

I used the [Google Analytics Debugger](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna/related) extension to confirm that the events were being recorded properly on the test site.

I pushed my commits to the develop branch, but couldn't create a Pull Request for just that work since @michiecat has some un-merged changes from June related to the podtrac link. Not sure of the best way to resolve this so asking here.
